cloudnet/src/main/java/de/dytanic/cloudnet/CloudNetTick.java

```diff
@@ -142,4 +124,22 @@ private void updateServiceLogs() {
         }
     }
 
+    private void tryStartService(ServiceInfoSnapshot serviceInfoSnapshot) {
+        if (serviceInfoSnapshot.getServiceId().getNodeUniqueId().equals(CloudNet.getInstance().getComponentName())) {
+            ICloudService cloudService = this.cloudNet.getCloudServiceManager().getCloudService(serviceInfoSnapshot.getServiceId().getUniqueId());
+            if (cloudService == null) {
+                // We found a ghost service (a service which has no CloudService but a still a ServiceInfoSnapshot)
```

Comment:
Can't we just fix that those ghost services exist?

Reply:
They should never exist tho. This is just a prevention..